### ISSUE TITLE
Potential Bug: Not clear why exit 2 to in github actions

### ISSUE THREAD

**Issue by Olgoetz** [terraform-linters/tflint#1761]
I leverage tflint github actions for code scanning.

The actions looks like this:
```yaml
name: TFLint
on:
  push:
    branches: [ master ]
  pull_request:

jobs:
  tflint:
    runs-on: self-hosted

    steps:
    - uses: actions/checkout@v3
      name: Checkout source code

    - uses: actions/cache@v3
      name: Cache plugin dir
      with:
        path: ~/.tflint.d/plugins
        key: self-hosted-tflint-${{ hashFiles('.tflint.hcl') }}

    - uses: terraform-linters/setup-tflint@v3
      name: Setup TFLint
      with:
        tflint_version: v0.44.1

    - name: Show version
      run: tflint --version

    - name: Init TFLint
      run: tflint --init
      env:
        # https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/plugins.md#avoiding-rate-limiting
        GITHUB_TOKEN: ${{ github.token }}

    - name: Run TFLint
      run: tflint -f compact

```
It is a super simple terraform code. But running through the checks, the tflint workflow fails with this:

```
Run tflint -f compact
2 issue(s) found:

Warning: main.tf:3[5](https://github.axa.com/mcm-orchestration/it-campus-day-pac-demo/runs/307061?check_suite_focus=true#step:7:6):1: Warning - Missing version constraint for provider "aws" in "required_providers" (terraform_required_providers)
:0:0: Warning - terraform "required_version" attribute is required (terraform_required_version)
Error: Process completed with exit code 2.
```

Why does it exit with an error although there are just warnings?

**Comment by bendrucker** [comment]
This is unrelated to GitHub Actions. See `--minimum-failure-severity`. By default any issues regardless of level cause a 2 exit code. See also: https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/config.md#force.

**Comment by Olgoetz** [comment]
Thank you @bendrucker . Much appreciated :)
